lib/internal/test_runner/runner.js

```diff
@@ -675,13 +676,13 @@ function run(options = kEmptyObject) {
     coverage,
     coverageExcludeGlobs,
     coverageIncludeGlobs,
+    coverageExcludeFiles: ArrayPrototypeMap(testFiles, (file) => resolve(cwd, file)),
```

Comment:
A) IMHO this can just be the testFiles list. The resolution doesn't really need to occur. IIUC the cwd here is the same as the coverage's CWD.
B) Along with A), this should probably be named `testFiles` for clarity

relativePath can be used, right?

Reply:
A) I was almost going to do so haha. The thing is that these files could come from the API (calling `run({ files })` which is already done in the tests) so I need to sanitize them as they can be absolute or relative, or even a complex path like `src/../lib/mytest.js`, so either way I need to iterate over them. I'm thinking that I need to add more tests, like "using weird paths". If I don't resolve the files, these tests are going to fail. I can iterate and get the absolute path or the relative path, but either way I need to iterate to sanitize.

B) The name was chosen based on the already existant filtering properties, as this is also a filtering property which applies inside the very same filtering function, but this is my opinion and I don't know what are the guidelines so I do not have any problem changing it.

Reply:
Ahh. I didn't think about the weird paths. Anyway, the name should probably be testFiles because that's a better description of what's actually being stored